src/include/network/services/routing.rb

```diff
@@ -140,6 +140,23 @@ def initialize_network_services_routing(include_target)
       }
     end
 
+    # Validates user's input obtained from Netmask field
+    #
+    # It currently allows to use netmask for IPv4 (e.g. 255.0.0.0) or
+    # prefix length. If prefix length is used it has to start with '/'.
+    # For IPv6 network, only prefix length is allowed.
+    #
+    # param [string] netmask or /<prefix length>
+    def valid_netmask?(netmask)
+      return false if netmask.nil? || netmask.empty?
+      return true if Netmask.Check4(netmask)
+
+      prefix = netmask[1..-1].to_i
+      return true if netmask.start_with?("/") && prefix.between?(1, 128)
```

Comment:
well, I think this code is more readable, as prefix is needed only if it start with "/"
```
if netmask.start_with?("/")
  return netmask[1..-1].to_i.between?(1, 128)
end

false
```